docs/commands/bundle.md

```diff
@@ -20,32 +20,23 @@ redocly bundle --version
+| apis                       | [string] | List of API description root filenames or names assigned in the `apis` section of your Redocly configuration file. Default values are all names defined in the `apis` section within your configuration file.                                                   |
+| --config                   | string   | Specify path to the [config file](#custom-configuration-file).                                                                                                                                                                                                  |
+| --dereferenced, -d         | boolean  | Generate fully dereferenced bundle.                                                                                                                                                                                                                             |
+| --ext                      | string   | Specify bundled file extension. Possible values are `json`, `yaml`, or `yml`. Default value is `yaml`.                                                                                                                                                          |
+| --extends (**Deprecated**) | [string] | Can be used in combination with `--lint` to [extend a specific configuration](./lint.md#extend-configuration). Default values are taken from the Redocly configuration file.                                                                                    |
+| --force, -f                | boolean  | Generate bundle output even when errors occur.                                                                                                                                                                                                                  |
+| --help                     | boolean  | Show help.                                                                                                                                                                                                                                                      |
+| --keep-url-references, -k  | boolean  | Keep absolute url references.                                                                                                                                                                                                                                   |
+| --lint-config              | string   | Specify the severity level for the configuration file. <br/> **Possible values:** `warn`, `error`, `off`. Default value is `warn`.                                                                                                                              |
```

Comment:
Is this still relevant?

Reply:
Yes, this `lint-config` settings is still relevant.
It is used for command wrapper checks.